recipe_scrapers/felixkitchen.py

```diff
@@ -0,0 +1,85 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._utils import normalize_string, unsupported_field_warning
+
+MARK_SEPARATOR = " "
+INGREDIENT_SEPARATOR = "• "
+
+
+class FelixKitchen(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "felix.kitchen"
+
+    def author(self):
+        return "FEL!X KITCHEN"
```

Comment:
From the HTML contents, I think the author credits themselves as `FEL!X`; let's retrieve this from the page (`div.entry-posted > div.posted-info`).